rocAL/source/pipeline/tensor.cpp

```diff
@@ -399,6 +447,25 @@ unsigned Tensor::copy_data(void *user_buffer, RocalOutputMemType external_mem_ty
     return 0;
 }
 
+unsigned Tensor::copy_data(void *user_buffer, uint max_y1, uint max_x1) {
```

Comment:
why there is a return type for this function? Are we doing any error handing based on return value of this function?

Reply:
We are handling it that way for other signatures of copy_data too. Maintaining the same here. When mem_handle is NULL - would return 0